index.html

```diff
@@ -1,45 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
+  <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Document</title>
+      <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
     <div class="wrapper">
-        <header>
-          <nav>
-              <a href="index.html" class="logo">Charles</a>
-              <img src="images/Union.png" alt="MenuBar">
-          </nav>
-        </header>
-        <section class="content">
-            <article>               
-                <h1>I'm Charles Nice to meet you!</h1>
-                <p>
-                    My Name is Okoye Charles and I’m a software developer!
-                     I can help you build a product ,
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and need help on a project, I am available
-                     for hire.
-                </p>
-            </article>
-        
-            <div class="social-header">
-                <p>LET'S CONNECT</p>
-            </div>
-            <nav class="social-links">
-                <ul>
-                    <li><a href="https://twitter.com/OkoyeCharles_"><img src="images/Vector.png" alt="Twitter Icon"></a></li>
+      <div class="home-wrap">
+        <section class="first-section">
+          <h1>.</h1>
+          <header>
+            <nav>
+                <a href="index.html" class="logo">Charles</a>
+                <img src="images/Union.png" alt="MenuBar">
+            </nav>
+          </header>
+          <section class="content">
+            <h1>I'm Charles Nice to meet you!</h1>
+              <article>               
+                <h2 id="invisible">.</h2>
+                  <p>
+                      My Name is Okoye Charles and I’m a software developer!
```

Comment:
- [ ] [OPTIONAL] Kindly reduce the spacing between the header and the  title of the pages as it will improve the look of your portfolio
<img width="359" alt="Screen Shot 2022-05-27 at 08 31 57" src="https://user-images.githubusercontent.com/28307960/170644317-f81e1790-ee7b-4b45-b85e-55f1d5e1523e.png">